README.md

```diff
@@ -1,6 +1,6 @@
 # wdio-browserstack-reporter
 
-The master branch contains reporter for **WebdriverIO 4**, for **WebdriverIO 5** please refer to [wdio-browserstack-reporter/wdio5](https://github.com/browserstack/wdio-browserstack-reporter/tree/wdio5)
+The branch contains reporter for **WebdriverIO 4**, for **WebdriverIO 5** and **WebdriverIO 6** please refer to [wdio-browserstack-reporter/wdio5](https://github.com/browserstack/wdio-browserstack-reporter/tree/wdio5)
 
 A WebdriverIO plugin which enables BrowserStack reports on CI servers
```

Comment:
peer dependencies and version bump here as well?  @harshitvasu

Reply:
Yes, had discussed with yash, he told it was a minor change to do that. I will do it as you mentioned it now